src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -392,4 +399,34 @@ default String clueNotifyMessage() {
         return "%USERNAME% has completed a %CLUE% clue, they have completed %COUNT%.\nThey obtained:\n\n%LOOT%";
     }
 
+    @ConfigItem(
+        keyName = "speedrunEnabled",
+        name = "Enable speedruns",
+        description = "Enable notifications for when you complete a speedrun",
+        position = 31,
+        section = speedrunSection
+    )
+    default boolean notifySpeedrun() {
+        return true;
+    }
+
+    @ConfigItem(
+        keyName = "speedrunPBMessage",
+        name = "PB message",
+        description = "%USERNAME% to insert your username, %QUEST% to insert the quest name, %TIME% to insert your new time",
+        position = 32,
+        section = speedrunSection
+    )
+    default String speedrunPBMessage() {
+        return "%USERNAME% has just beat their in a speedrun of %QUEST% in %TIME%";
+    }
+
+    @ConfigItem(
+        keyName = "speedrunSendImage",
+        name = "Send Image",
+        description = "Send image with the notification",
+        position = 33,
+        section = clueSection
+    )
+    default boolean speedrunSendImage() { return true; }
```

Comment:
re-order these 2 so it follows the order implemented in #17

Reply:
![image](https://user-images.githubusercontent.com/25011746/196004034-ac087449-60e5-4abc-a405-547773f7661e.png)
fixed in 8731d59